### PR TITLE
updated docusaurus.config

### PR DESCRIPTION
**Pull request by Veer0x1**

### CODE REVIEW

**Comment by eshantri** [comment]
Hey @Veer0x1 please provide a description of what this PR does, also please check why build test is failing?
cc @mdietze

File: docusaurus.config.js [hunk 1]
```diff
@@ -94,7 +94,8 @@ module.exports = {
       ],
       copyright: `Copyright © ${new Date().getFullYear()} PEcAn Project.`,
     },
-    hideableSidebar: true,
+
+    docs: { sidebar: { hideable: true } },
   },
```
Comment (eshantri) [inline docusaurus.config.js:99]: Please add a comment to explain this line.

**Review by eshantri — CHANGES_REQUESTED** [review]

**Comment by Veer0x1** [comment]
By enabling the themeConfig.docs.sidebar.hideable option, you can make the entire sidebar hideable, allowing users to better focus on the content. This is especially useful when content is consumed on medium-sized screens (e.g. tablets).

File: docusaurus.config.js (hunk 1 above)
Comment (Veer0x1) [reply docusaurus.config.js:99]: Screenshot 2023-03-08 at 12 30 47 PM
When I was trying to set up the project locally, I got this error so I changed the config and it starts working fine.

Comment (Veer0x1) [reply docusaurus.config.js:99]: Screenshot 2023-03-08 at 12 31 16 PM
But when I made PR this showed this error. How can I fix this?

Comment (Veer0x1) [reply docusaurus.config.js:99]: it also works when I comment out line 97

**Pull request closed without merging**